localstack/services/sqs/sqs_listener.py

```diff
@@ -214,15 +214,15 @@ def _process_sent_message(path: str, req_data: Dict[str, str], headers: Dict, re
 
     # extract data from XML response - assume data is wrapped in 2 parent elements
     response_data = xmltodict.parse(response.content)
-    response_data = list(response_data.values())[0]
-    response_data = list(response_data.values())[0]
```

Comment:
Thanks for fixing this @rlodge !

nitpick: We could also make the sub-element selection generic, something like this: 
```
response_data = response_data[f"{action}Response"][f"{action}Result"]
```
(shouldn't hold back the merge, though - we can pick this up in a future iteration..). Thanks!